apps/settings/js/panels/homescreen_details/homescreen_cols.js

```diff
@@ -1,34 +1,49 @@
-define(function(require) {
+/**
+ * Manage the column layout of the legacy vertical home screen.
+ */
+define(require => {
```

Comment:
No one want to maintain a part of code forever by himself. We'd better align coding style with other modules, so it's easier to read for other maintainers.